scripts/exsrw_aqm_lbcs.sh

```diff
@@ -8,7 +8,11 @@
 #-----------------------------------------------------------------------
 #
 . ${USHsrw}/source_util_funcs.sh
-source_config_for_task "task_get_extrn_lbcs|task_make_orog|task_make_lbcs|cpl_aqm_parm|task_aqm_lbcs" ${GLOBAL_VAR_DEFNS_FP}
+for sect in user nco platform workflow nco global verification cpl_aqm_parm \
+  constants fixed_files grid_params \
+  task_aqm_lbcs task_get_extrn_lbcs task_make_lbcs task_make_orog ; do
```

Comment:
There is no `task_aqm_lbcs` in `config_defaults.yaml`.

```suggestion
  task_get_extrn_lbcs task_make_lbcs task_make_orog ; do
```